Functions/Functions/User/GetUserFunction.cs

```diff
@@ -0,0 +1,80 @@
+using System;
+using System.IO;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.Azure.WebJobs;
+using Microsoft.Azure.WebJobs.Extensions.Http;
+using Microsoft.AspNetCore.Http;
+using Microsoft.Extensions.Logging;
+using Newtonsoft.Json;
+using OidcApiAuthorization.Abstractions;
+using OidcApiAuthorization.Models;
+using System.Linq;
+
+namespace SSW.Rules.Functions
+{
+    public class GetUserFunction
+    {
+        private readonly RulesDbContext _dbContext;
+        private readonly IApiAuthorization _apiAuthorization;
+
+        public GetUserFunction(RulesDbContext dbContext, IApiAuthorization apiAuthorization)
+        {
+            _dbContext = dbContext;
+            _apiAuthorization = apiAuthorization;
+        }
+
+        [FunctionName("GetUserFunction")]
+        public async Task<IActionResult> Run(

```

Comment:
Please comment to explain what this function/class does. I am not clear on what it is supposed to do, a summary comment would help

Reply:
As per Matt W's recommendation please remove the summary comment. Sorry for misleading you! Also I'm not sure function is needed in the name since that is implied.

Reply:
Function is still in the name of this function, I think it should be consistent with the other functions so we should remove that part of the name.

Reply:
@bradystroud has mentioned that this is the standard we follow so he is going to change RemoveUserCommentsAccount to have function postfixed as well.

Reply:
I have made all function names end with Function to be consistent